src/mpyl/steps/run_properties.py

```diff
@@ -16,53 +15,42 @@ def construct_run_properties(
     config: dict,
     properties: dict,
     cli_parameters: MpylCliParameters = MpylCliParameters(),
-    run_plan: Optional[RunPlan] = None,
```

Comment:
These two parameters are only ever used for test purposes, and add a lot of complexity to the original function. These were replaced by a new [`stub_run_properties()` test function](https://github.com/Vandebron/gh-mpyl/pull/38/files#diff-98e5a744692c47a2036490c9579b543bcd95c23f71a5250db07de7cf041f60c8R22-R37).